OpenRA.Mods.Common/Widgets/Logic/PerfDebugLogic.cs

```diff
@@ -40,7 +41,13 @@ public PerfDebugLogic(Widget widget)
 					fpsReferenceFrame = Game.RenderFrame;
 				}
 
-				return $"FPS: {fps}\nTick {Game.LocalTick} @ {PerfHistory.Items["tick_time"].Average(Game.Settings.Debug.Samples):F1} ms\nRender {Game.RenderFrame} @ {PerfHistory.Items["render"].Average(Game.Settings.Debug.Samples):F1} ms\nBatches: {PerfHistory.Items["batches"].LastValue}";
+				var wfbSize = Game.Renderer.WorldFrameBufferSize;
+				var viewportSize = worldRenderer.Viewport.Rectangle.Size;
+				return $"FPS: {fps}\nTick {Game.LocalTick} @ {PerfHistory.Items["tick_time"].Average(Game.Settings.Debug.Samples):F1} ms\n" +
+					$"Render {Game.RenderFrame} @ {PerfHistory.Items["render"].Average(Game.Settings.Debug.Samples):F1} ms\n" +
+					$"Batches: {PerfHistory.Items["batches"].LastValue}\n" +
+					$"Viewport Size: {viewportSize.Width}, {viewportSize.Height} / {Game.Renderer.WorldDownscaleFactor}\n" +
```

Comment:
Can use `1920 x 1080` here instead of `1920, 1080` make this a little more obvious.